extensions/typescript/src/features/referenceProvider.ts

```diff
@@ -14,7 +14,7 @@ export default class TypeScriptReferenceSupport implements ReferenceProvider {
 	public constructor(
 		private client: ITypescriptServiceClient) { }
 
-	public provideReferences(document: TextDocument, position: Position, options: { includeDeclaration: boolean }, token: CancellationToken): Promise<Location[]> {
+	public provideReferences(document: TextDocument, position: Position, options: { includeDeclaration: boolean }, _: (value: Location) => void, token: CancellationToken): Promise<Location[]> {
```

Comment:
Assuming the `_` param is to get around TypeScript's unused parameters warning?

If so, we should find a better solution. Otherwise it would require updating a fair bit of code in extensions to make them compile, so it'd be pretty backward-incompatible.